src/components/TrafimageMaps/TrafimageMaps.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../globals.scss';
 
 .tm-trafimage-maps {
-  font-family: 'SBBWeb Roman', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-family: 'SBBWeb-Roman', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
In the ticket it's written ,' Arial as fallback'

Reply:
I changed with a suggestion! Good catch